pkg/report/writer.go

```diff
@@ -91,6 +92,8 @@ func Write(report types.Report, option flag.Options) error {
 		return xerrors.Errorf("unknown format: %v", option.Format)
 	}
 
+	report.CreatedAt = clock.Now().Unix()
```

Comment:
I think it's more straightforward to add `CreatedAt` here.
https://github.com/aquasecurity/trivy/blob/d19c7d9f292759848aa77109357b405a64716c78/pkg/scanner/scan.go#L173-L189

Reply:
@knqyf263 Thanks suggestion. I've fixed it.